packages/styleguide/src/styles/expo-theme.css

```
@@ -19,6 +19,20 @@
   --expo-color-white: #fff;
   --expo-color-black: #000;
 
+  /* App Backgrounds */
```

Some more specificity to make it easier to understand where this is used:

```suggestion
  /* Default App Icon Backgrounds */
```